src/olympia/addons/models.py

```diff
@@ -2169,6 +2169,14 @@ class ReplacementAddon(ModelBase):
     class Meta:
         db_table = 'replacement_addons'
 
+    @staticmethod
+    def is_external_path(path):
+        scheme = path.split('://')[0].lower()
+        return scheme in ['http', 'https']
+
+    def is_external(self):
```

Comment:
This name is not obvious when used elsewhere.

Reply:
Alternate name?

Reply:
`is_external_url` which would be kinda weird since we're talking about `path` everywhere, uhm... but in the end imho it's a valid name. I think I vote for that.